include/boost/astronomy/io/hdu.hpp

```diff
@@ -217,7 +168,17 @@ struct hdu
     */
     std::size_t naxis(std::size_t n = 0) const
     {
-        return this->naxis_[n];
+        return this->naxis_[n - 1]; // 1 Based indexing
```

Comment:
Users don't have to worry about 0 based indexing but I don't thing this should it be done